bower.json

```diff
@@ -40,6 +40,6 @@
     "angular-route": "1.3.0 - 1.4.*",
     "angular-bootstrap": "0.13.x",
     "lodash": "3.x",
-    "patternfly": "2.2.0"
+    "patternfly": "https://github.com/patternfly/patternfly.git#41b81ba31dba9b451dc5b57b17a37b8d5d37d3af"
```

Comment:
The current toolbar styling in Patternfly has not yet been released. As I put in the PR notes, we will need to update this once PF releases and before we release the next version of angular-patternfly